Model/UrlInformation.php

```diff
@@ -0,0 +1,228 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2014 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Bundle\SeoBundle\Model;
+
+/**
+ * @author Maximilian Berghoff <Maximilian.Berghoff@gmx.de>
+ */
+class UrlInformation
+{
+    /**
+     * Decides whether the content is visible on sitemap.
+     *
+     * @var bool
+     */
+    private $visible;
+
+    /**
+     * @var string
+     */
+    private $loc;
+
+    /**
+     * @var \DateTime
+     */
+    private $lastmod;
+
+    /**
+     * @var string One of the official/allowed.
+     */
+    private $changeFreq;
+
+    /**
+     * @var float
+     */
+    private $priority;
+
+    /**
+     * @var array
+     */
+    private $allowedChangeFreqs = array('always', 'hourly', 'daily', 'weekly', 'monthly', 'yearly', 'never');
+
+    /**
+     * @var string $label As a string to display the route i.e. in html views.
+     */
+    private $label;
+
+    /**
+     * @var array|AlternateLocale[]
+     */
+    private $alternateLocales;
+
+    public function __construct()
+    {
+        $this->alternateLocales = array();
+    }
+
+    /**
+     * @return boolean
+     */
+    public function isVisible()
+    {
+        return $this->visible;
+    }
+
+    /**
+     * @param boolean $visible
+     */
+    public function setVisible($visible)
+    {
+        $this->visible = $visible;
+    }
+
+    public function toArray()
+    {
+        $result = array(
+            'loc'               => $this->loc,
+            'label'             => $this->label,
+            'changefreq'        => $this->changeFreq,
+            'lastmod'           => $this->lastmod,
+            'priority'          => $this->priority,
+            'alternate_locales' => array()
+        );
+
+        foreach ($this->alternateLocales as $locale) {
+            $result['alternate_locales'][] = array('href' => $locale->href, 'href_locale' => $locale->hrefLocale);
+        }
+
+        return $result;
+    }
+
+    /**
+     * @return string
+     */
+    public function getChangeFreq()
+    {
+        return $this->changeFreq;
+    }
+
+    /**
+     * @param string $changeFreq One of the official/allowed ones.
+     *
+     * @return $this
+     */
+    public function setChangeFreq($changeFreq)
+    {
+        if (!in_array($changeFreq, $this->allowedChangeFreqs)) {
+            return $this;
```

Comment:
Silently ignoring things makes debugging very hard. We should throw an exception here imo